.github/workflows/merge-develop-to-stage.yml

```diff
@@ -43,6 +43,27 @@ jobs:
       CYPRESS_ADMIN_PASSWORD: ${{ secrets.CYPRESS_ADMIN_PASSWORD }}
       CYPRESS_ADMIN_USER: ${{ secrets.CYPRESS_ADMIN_USER }}
 
+  deploy-swb-reference-byon:
```

Comment:
This will deploy along with [deploy-swb-reference](https://github.com/aws-solutions/solution-spark-on-aws/pull/722/files#diff-0cf332403f4e6e08877489fcd34a6b3a606ad577c03d7e1bb5e15d5aaf78fc5dL35). Please make sure this is deployed in a different region, if using the same account to avoid conflicts during deployment

Reply:
Taken care of [here](https://github.com/aws-solutions/solution-spark-on-aws/blob/899acd067f046ba8c90767c0acfae12a27a5effb/.github/workflows/merge-develop-to-stage.yml#L52)